app/models/account.rb

```diff
@@ -176,6 +186,7 @@ def validate_against_product(product, user)
     return "The #{self.type_string} has insufficient price groups" unless product.can_purchase?((self.price_groups + user.price_groups).flatten.uniq.collect {|pg| pg.id})
 
     # check chart string account number
+    # TODO: should this include SplitAccount?
```

Comment:
Possibly, it should check against each component of the split account. Seems like this could be a method call that in `Account` does nothing. In `NufsAccount`, it does this. In `SplitAccount`, it loops through each of the sub-accounts and calls the method on that account.

This might be something to chat with Matt about. I don't remember exactly how "account is open" really works.